content/en/tracing/setup/istio.md

```diff
@@ -30,21 +30,80 @@ Datadog APM is available for Istio v1.1.3+ on Kubernetes clusters.
 
 ### Istio Configuration and Installation
 
-To enable Datadog APM, a [custom Istio installation][3] is required to add two extra options before Istio is installed. These options are passed at the final `helm template` or `helm install` step:
+To enable Datadog APM, a [custom Istio installation][3] is required to set two extra options when installing Istio.
 
-- `--set pilot.traceSampling=100.0`
-- `--set global.proxy.tracer=datadog`
+- `--set values.global.proxy.tracer=datadog`
+- `--set values.pilot.traceSampling=100.0`
 
-Eg: an installation using the `default` configuration profile would use the following command:
+```shell
+istioctl manifest apply --set values.global.proxy.tracer=datadog --set values.pilot.traceSampling=100.0
+```
+
+Traces will be generated when the namespace for the pod has sidecar injection enabled. This is done by adding
+the `istio-injection=enabled` label.
 
 ```shell
-helm template install/kubernetes/helm/istio --name istio --namespace istio-system --set pilot.traceSampling=100.0 --set global.proxy.tracer=datadog | kubectl apply -f -
+kubectl label namespace example-ns istio-injection=enabled
+```
+
+Traces are generated when Istio is able to determine the traffic is using an HTTP-based protocol.
+By default, it will try to automatically detect this. It can be manually configured by naming the ports in your
+application's deployment and service. More information can be found in Istio's documentation for [Protocol Selection][4]
+
+By default, the service name used when creating traces is generated from the deployment name and namespace. This can be
+set manually by adding an `app` label to the deployment's pod template.
+For [CronJobs][5], the `app` label should be added to the job template, as the generated name comes from the `Job` instead
+of the higher-level `CronJob`.
+
+The environment for traces can be set on a per-deployment basis using the `apm.datadoghq.com/env` annotation.
+```yaml
+    metadata:
+      annotations:
+        apm.datadoghq.com/env: '{ "DD_ENV": "test" }'
+```
+
+### Running Agent as Deployment and Service
+
+If the Agents on your cluster are running as a Deployment and Service instead of the default DaemonSet, then an additional option is required, to specify the DNS address and port of the Agent.
```

Comment:
```suggestion
If the Agents on your cluster are running as a Deployment and Service instead of the default DaemonSet, then an additional option is required to specify the DNS address and port of the Agent.
```